private/util/content_md5.go

```diff
@@ -0,0 +1,78 @@
+package util
```

Comment:
This package would probably be better named more specifically like `private/checksum`